tests/acceptance/features/bootstrap/TUSContext.php

```diff
@@ -407,7 +436,9 @@ public function userUploadsFileWithChecksum(
 		string $offset,
 		string $content
 	): void {
-		$this->sendsAChunkToTUSLocationWithOffsetAndData($user, $offset, $content, $checksum);
+		$response = $this->sendsAChunkToTUSLocationWithOffsetAndData($user, $offset, $content, $checksum);
+		$this->featureContext->setResponse($response);
+		WebDavHelper::$SPACE_ID_FROM_OCIS = '';
```

Comment:
was it necessary in these methods? Maybe it could be great to have it in cleanup methods (hooks)

Reply:
```

         /**
	 * @AfterScenario
	 *
	 * @return void
	 *
	 * @throws Exception
	 */
	public function cleanSpaceIDFromOcisAfterTest(): void {
		WebDavHelper::$SPACE_ID_FROM_OCIS = '';
	}


```

can i make this after hook then

Reply:
> was it necessary in these methods? Maybe it could be great to have it in cleanup methods (hooks)
previously sendsAChunkToTUSLocationWithOffsetAndData() function used WebDavHelper::$SPACE_ID_FROM_OCIS = '' after setting  but since i changed it to function to return response so i also reset $SPACE_ID_FROM_OCIS where the function is used.

Reply:
maybe put it in user cleanup hook